src/conntrackd/conntrackd.conf.j2

```diff
@@ -0,0 +1,52 @@
+Sync {
+        Mode NOTRACK {
+                DisableInternalCache on
+                DisableExternalCache on
+                StartupResync on
+        }
+        Multicast {
+                IPv4_address 225.0.0.50
+                Group 3780
+		IPv4_interface {{ cluster_ip_addr }}
+                Interface team0
+                SndSocketBuffer 1249280
+                RcvSocketBuffer 1249280
+                Checksum on
+        }
+        Options {
+                TCPWindowTracking Off
+                ExpectationSync On
+        }
+}
+General {
+        Systemd on
+        HashSize 32768
+        HashLimit 131072
+        Syslog on
+        LockFile /var/lock/conntrackd.lock
+        UNIX {
+                Path /var/run/conntrackd.sock
+        }
+        NetlinkBufferSize 2097152
+        NetlinkBufferSizeMaxGrowth 8388608
+        NetlinkOverrunResync On
+        NetlinkEventsReliable Off
+        EventIterationLimit 100
+        Filter From Kernelspace {
+                Protocol Accept {
+                        TCP
+                }
+                Address Accept {
+                        {% if conntrackd_ip_list is defined -%}
```

Comment:
Is there really a need to check for that ?
This file should not be uploaded if the variable `conntrackd_ip_list` doesn't exists.

Reply:
that's fair, will change